git-cliff-core/src/commit.rs

```diff
@@ -459,17 +466,35 @@ impl Serialize for Commit<'_> {
 		commit.serialize_field("merge_commit", &self.merge_commit)?;
 		commit.serialize_field("extra", &self.extra)?;
 		#[cfg(feature = "github")]
-		commit.serialize_field("github", &self.github)?;
+		serialize_remote::<S>(&mut commit, "github", &self.github)?;
 		#[cfg(feature = "gitlab")]
-		commit.serialize_field("gitlab", &self.gitlab)?;
+		serialize_remote::<S>(&mut commit, "gitlab", &self.gitlab)?;
 		#[cfg(feature = "gitea")]
-		commit.serialize_field("gitea", &self.gitea)?;
+		serialize_remote::<S>(&mut commit, "gitea", &self.gitea)?;
 		#[cfg(feature = "bitbucket")]
-		commit.serialize_field("bitbucket", &self.bitbucket)?;
+		serialize_remote::<S>(&mut commit, "bitbucket", &self.bitbucket)?;
+		if let Some(remote) = &self.remote {
+			commit.serialize_field("remote", remote)?;
+		}
 		commit.end()
 	}
 }
 
+fn serialize_remote<S>(
```

Comment:
Why do we need this method? Also, can you document it? 😊

Reply:
removed in https://github.com/orhun/git-cliff/pull/822/commits/04e62efc17772d4e20809e98b332f4ff678fb740 as it was no longer needed 👍
Initially I didn't know you wanted git cliff commit to move to `commit.remote` so I thought I had to fill `commit.remote` with `commit.github` info. That's why I needed the method 👍